maxim/logger/parsers/generation_parser.py

```
@@ -163,9 +178,9 @@ def parse_model_parameters(parameters: Optional[Dict[str, Any]]) -> Dict[str, An
             continue
         if not isinstance(value, str):
             try:
-                new_parameters[key] = json.dumps(value)
+                new_parameters[key] = json.dumps(value, default=default_json_serializer)
             except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)